### PR TITLE
PR 110: new default

### PR DESCRIPTION
**Pull request by uhop**
This PR includes the code, the tests, and the docs. Incidentally, the lock file was out of sync with the package file, so it is trivially updated too.

It is built on PR #110 according to the comment https://github.com/koajs/compress/pull/110#issuecomment-645489580 as mentioned in #112

### CODE REVIEW

**Comment by niftylettuce** [comment]
I merged the wrong PR

**Comment by niftylettuce** [comment]
v5.0.0 published to npm and GitHub

https://github.com/koajs/compress/releases/tag/v5.0.0

File: __tests__/index.js [hunk 1]
```diff
@@ -245,16 +245,55 @@ describe('Compress', () => {
...
+      ctx.type = 'text'
+      ctx.body = buffer
+    })
+    server = app.listen()
 
-    app.use(compress())
-    app.use(sendBuffer)
+    request(server)
+      .get('/')
+      .set('Accept-Encoding', '')
```
Comment (dobesv) [inline __tests__/index.js:261]: Ah interesting workaround.

Comment (uhop) [reply __tests__/index.js:261]: `unset()` didn't work, but setting the empty value did the trick.

**Pull request merged**